qa/tasks/cephfs/mount.py

```diff
@@ -124,48 +146,46 @@ def _setup_brx_and_nat(self):
         if p == False:
             raise RuntimeError("No default gw found")
         gw = p[0].split()[7]
-        args = ["sudo", "bash", "-c",
-                "iptables -A FORWARD -o {0} -i ceph-brx -j ACCEPT".format(gw)]
+        args = ['sudo', 'iptables', '-A', 'FORWARD', '-o', '{0}'.format(gw), '-i', 'ceph-brx', '-j', 'ACCEPT']
         self.client_remote.run(args=args, timeout=(5*60), omit_sudo=False)
```

Comment:
While you're here fixing this, why not do:

````
self.run_shell_payload(f"""
sudo iptables ...
sudo iptables ...
""", omit_sudo=False)
````

Reply:
Cool, I didn't notice this helper, will fix it.
Thanks.